src/mumble/MumbleAPI_structs.h

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021-2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+/// This header file includes the latest definition of Mumble's API
+/// and also all older API structs for backward compatibility.
+
+#ifndef EXTERNAL_MUMBLE_PLUGIN_API_LATEST_H_
```

Comment:
The include guard still has to be adapted to `MUMBLE_MUMBLE_PLUGIN_API_STRUCTS_H_`

Reply:
Fixed.